SFSymbol2/SFSymbol2.0.swift

```diff
@@ -0,0 +1,811 @@
+//
+//  SFSymbol2.0.swift
```

Comment:
Good job so far! Just that this file name should be something like `SFSymbol2.swift`